sim-cli/src/main.rs

```diff
@@ -1,7 +1,9 @@
+use std::collections::HashMap;
 use std::path::PathBuf;
+use std::sync::Arc;
```

Comment:
Add 
```
use std::str::FromStr;
use bitcoin::secp256k1::PublicKey;
```

Reply:
You'll need to also import `bitcoin` to this crate

Reply:
ack